packages/theme/src/theme/_forms.scss

```diff
@@ -349,6 +402,27 @@ form {
 		select {
 			padding: 0;
 		}
+
+		&.multi-select-readonly {
```

Comment:
What is this class ?

Reply:
This it's not related to Bootstrap indeed